runtime/lua/vim/lsp/client.lua

```diff
@@ -840,7 +840,7 @@ end
 --- Get options for a method that is registered dynamically.
 --- @param method vim.lsp.protocol.Method
 function Client:_supports_registration(method)
-  local capability_path = lsp.protocol._request_name_to_client_capability[method]
+  local capability_path = lsp.protocol._request_name_to_client_capability[method] or {}
```

Comment:
I'm unsure of whether we should return `false` here though, as we return `true` with unknown methods in `supports_method`:
https://github.com/MariaSolOs/neovim/blob/26fcefe000b2df3e2a2c9e54f6219ce81c021ebb/runtime/lua/vim/lsp/client.lua#L1074-L1077

Reply:
But I guess custom methods aren’t usually dynamically registered…